tests/tck/features/match/WithUnwind.feature

```diff
@@ -0,0 +1,140 @@
+Feature: With clause and Unwind clause
+
+  Background:
+    Given a graph with space named "nba"
+
+  Scenario: with return
+    When executing query:
+      """
+      WITH [1, 2, 3] AS a, "hello" AS b
+      RETURN a, b
+      """
+    Then the result should be, in any order:
+      | a       | b       |
+      | [1,2,3] | "hello" |
+    When executing query:
+      """
+      WITH [1, 2, 3] AS a WITH a AS a, "hello" AS b
+      RETURN a, b
+      """
+    Then the result should be, in any order, with relax comparison:
+      | a       | b       |
+      | [1,2,3] | "hello" |
+
+  @skip
```

Comment:
We haven't support with and match combination now ?

Reply:
Yes, they will be in another PR.